src/menu/views/settings_editor.c

```diff
@@ -52,18 +151,31 @@ static void draw (menu_t *menu, surface_t *d) {
         format_switch(menu->settings.rumble_enabled)
     );
 
+
     component_actions_bar_text_draw(
         ALIGN_LEFT, VALIGN_TOP,
-        "\n"
+        "A: Change\n"
         "B: Back"
     );
 
+    if (settings_changed) {
+        component_actions_bar_text_draw(
+                ALIGN_RIGHT, VALIGN_TOP,
+                "\n"
+                "R: Save"
+            );
+    }
```

Comment:
this will be 100% confusing to the end user, just update the settings file when single option is changed instead of hoping that user will remember to press the save button